pkg/shell/credentials.js

```diff
@@ -255,12 +256,14 @@ define([
                     dfd.resolve();
                 })
                 .fail(function(ex) {
```

Comment:
I think the failure method can have partial data that wasn't consumed by stream, see [docs](http://cockpit-project.org/guide/latest/cockpit-spawn.html#cockpit-spawn-fail).